vmdk_plugin/drivers/photon/photon_driver.go

```diff
@@ -54,6 +55,8 @@ const (
 	fsTypeTag            = "Fs_Type"
 )
 
+var fullVolumeNameMap map[string]string // save full volume names to avoid trip to vmdkops service
```

Comment:
it is 'mount id -> fullVOlumeName', right ? Or is is something else  
Please add comment ("mountId->fullVolumeName map" or whatever it is)